### DISCUSSION TITLE
Disable camera through MQTT

### DISCUSSION THREAD

**Discussion by AghilasMessara** [blakeblackshear/frigate#7692] [Ideas]
Hello everyone,
I have some cameras that I use only when we are away for a long period of time. I have no need to record my living room when we are home 😆 
We can already disable pretty much every processing done on the camera (motion, detector...), but I want to completely disable it just like we can do it from the configuration file. The point is that there is no need to use resources if I don't need it. In fact these cameras are even unplugged most of the time.
What do you think about this?

**Comment by NickM-27** [comment]
#1911
